README.md

```diff
@@ -14,7 +14,7 @@ Make sure you have a nightly Rust installed. Something like this:
 
 ```bash
 $ rustc --version
-rustc 1.13.0-nightly (267cde259 2016-05-25)
+rustc 1.10.0-nightly (267cde259 2016-05-25)
```

Comment:
Yes, @WilsonGiese is right: we used to be tied to a specific version, but with the recent changes to `xargo`, we're not any more.